client/utils/test/account-fees.tsx

```diff
@@ -265,4 +275,91 @@ describe( 'Account fees utility functions', () => {
 			);
 		} );
 	} );
+
+	describe( 'formatMethodFeesTooltip()', () => {
+		beforeAll( () => {
+			global.wcpaySettings = { connect: { country: 'US' } };
+		} );
+		afterAll( () => {
+			global.wcpaySettings = { connect: { country: '' } };
+		} );
+
+		it( 'displays base percentage and fixed fee - no custom fee nor discount', () => {
+			const methodFees = mockAccountFees( {
+				percentage_rate: 0.123,
+				fixed_rate: 456.78,
+				currency: 'USD',
+			} );
+
+			methodFees.additional = {
+				percentage_rate: 0.01,
+				fixed_rate: 0,
+				currency: 'USD',
+			};
+
+			methodFees.fx = {
+				percentage_rate: 0.01,
+				fixed_rate: 0,
+				currency: 'USD',
+			};
+
+			expect(
```

Comment:
Nice! But I'd use `container` instead of `baseElement`. Quoting the [docs](https://testing-library.com/docs/react-testing-library/api#baseelement-1):

> This is useful when the component you want to test renders something outside the container div, e.g. when you want to snapshot test your portal component which renders its HTML directly in the body.

I don't believe this is our case, so it seems [container](https://testing-library.com/docs/react-testing-library/api#container-1) is a better fit.

And what do you think about the following?:
```suggestion
			const { container } = render(
				formatMethodFeesTooltip( methodFees )
			);

			expect( container ).toMatchSnapshot();
```

Reply:
Notes to myself - Read the docs of the libraries too!  Thank you Miguel 🙌 

Fixed this here d687bae